mypy.ini

```
@@ -617,6 +617,9 @@ ignore_errors = True
 [mypy-scipy.special._basic]
 ignore_errors = True
 
+[mypy-scipy.special._specfun2]
```

I think you actually need this. The ignore needs to be in the place which imports the untyped compiled code.

```suggestion
[mypy-scipy.special.__init__]
```

---

Hmm nevermind that hasn't worked...

---

Somehow I've fixed the floating-point errors but not the mypy error! I'll write about the floating-point errors below, but if you (or anyone) has any thoughts on the mypy error that would be great.